<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="TestLogin(failed)">
    <classes>
      <class name="Establishment.Login.testCase.TestCase_LoginFileUsernameCorrect_PasswordWrong">
        <methods>
          <include name="beforeMethod"/>
          <include name="f"/>
          <include name="afterMethod"/>
        </methods>
      </class> <!-- Establishment.Login.testCase.TestCase_LoginFileUsernameCorrect_PasswordWrong -->
      <class name="Establishment.Login.testCase.TestCase_LoginWrong">
        <methods>
          <include name="beforeMethod"/>
          <include name="f"/>
          <include name="afterMethod"/>
        </methods>
      </class> <!-- Establishment.Login.testCase.TestCase_LoginWrong -->
      <class name="Establishment.Login.testCase.TestCase_LoginFileUsernameWrong_PasswordCorrect">
        <methods>
          <include name="beforeMethod"/>
          <include name="f"/>
          <include name="afterMethod"/>
        </methods>
      </class> <!-- Establishment.Login.testCase.TestCase_LoginFileUsernameWrong_PasswordCorrect -->
    </classes>
  </test> <!-- TestLogin(failed) -->
</suite> <!-- Failed suite [Suite] -->
